WooCommerce/WooCommerceUITests/Tests/StatsTests.swift

```diff
@@ -14,7 +14,8 @@ final class StatsTests: XCTestCase {
         try LoginFlow.login()
     }
 
-    func test_load_stats_screen() throws {
+    /// TODO: Update tests after the new stats screen is released.
+    func skipped_test_load_stats_screen() throws {
```

Comment:
@jostnes Just FYI we are disabling the outdated UI tests for dashboard stats since we've updated the UI for this feature. More details on the new UI design here: pe5sF9-2DL-p2